app/routes/events/components/EventDetail/index.js

```diff
@@ -186,85 +191,72 @@ export default class EventDetail extends Component<Props> {
       'hours'
     );
 
-    const infoItems: Array<?{ key: string, value: Node }> = [
-      event.company && {
-        key: 'Arrangerende bedrift',
-        value: (
-          <Link to={`/companies/${event.company.id}`}>
-            {event.company.name}
-          </Link>
-        ),
-      },
-      event.createdBy && {
-        key: 'Forfatter',
-        value: (
-          <Link to={`/users/${event.createdBy.username}`}>
-            {event.createdBy.fullName}
-          </Link>
-        ),
-      },
-      event.responsibleGroup && {
-        key: 'Arrangør',
-        value: (
-          <span>
-            {event.responsibleGroup.name}{' '}
-            {event.responsibleGroup.contactEmail && (
-              <a href={`mailto:${event.responsibleGroup.contactEmail}`}>
-                {event.responsibleGroup.contactEmail}
-              </a>
-            )}
-          </span>
-        ),
-      },
-      {
-        key: 'Hva',
-        value: eventTypeToString(event.eventType),
-      },
-      {
-        key: 'Når',
-        value: <FromToTime from={event.startTime} to={event.endTime} />,
-      },
-      { key: 'Finner sted i', value: event.location },
+    const deadlines: Array<?{ key: string, value: Node }> = [
       event.activationTime
         ? {
             key: 'Påmelding åpner',
-            value: <FormatTime time={eventRegistrationTime} />,
-          }
-        : null,
-      event.registrationDeadlineHours &&
-      !['OPEN', 'TBA'].includes(event.eventStatusType)
-        ? {
-            value: <FormatTime time={registrationCloseTime(event)} />,
-            key: 'Påmelding stenger',
-          }
-        : null,
-      event.unregistrationDeadlineHours &&
-      !['OPEN', 'TBA'].includes(event.eventStatusType)
-        ? {
-            value: <FormatTime time={unregistrationCloseTime(event)} />,
-            key: 'Avregistrering stenger',
+            value: (
+              <FormatTime
+                format="dd DD. MMM HH:mm"
+                time={eventRegistrationTime}
+              />
+            ),
           }
         : null,
       event.heedPenalties &&
       event.unregistrationDeadline &&
       !['OPEN', 'TBA'].includes(event.eventStatusType)
         ? {
-            key: 'Avregistreringsfrist',
-            value: <FormatTime time={event.unregistrationDeadline} />,
+            key: 'Frist for prikk',
+            value: (
+              <FormatTime
+                format="dd DD. MMM HH:mm"
+                time={event.unregistrationDeadline}
+              />
+            ),
```

Comment:
Could it be possible to add a tooltip here with something along the lines of  
`Lurer du på hvordan prikksystemet funker? Sjekk ut <a>arrangementsreglene</a>.`

If I didn't know what _prikk_ was this sentence might not make too much sense to me, and it feels like an intuitive place to let users know how the system works.

Reply:
I was just about to comment this - I vouch for it!

Reply:
I very much agree. Will add!